src/main/java/com/ungmydieu/todolist/models/Task.java

```diff
@@ -0,0 +1,29 @@
+package com.ungmydieu.todolist.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class Task {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+
+//    @Column(columnDefinition = "NOT NULL")
+    private String title;
+
+    private String description;
+
+    private Date startTime;
+
+    private Date endTime;
+
+    private boolean status;
```

Comment:
property status này a chưa thấy dùng ở đâu nhiều

Reply:
Cột này thật ra không thực sự cần thiết bởi vì ngữ nghĩa của nó có thể được suy ra từ `endTime`.

- endTime == null --> Chưa done
- endTime != null --> Đã done.

Cũng vì thế nên nếu mình thấy 1 record mà có `endTime` != null nhưng status = `Not done` thì không rõ thế nào, cái này sẽ tạo nên một sự thiếu nhất quán về dữ liệu.

Reply:
Done